optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -102,19 +177,34 @@ export const PreferentialTrials: FC<{ studyDetail: StudyDetail | null }> = ({
       }
     })
   }
+  const shuffleFlag = shuffleTrial + 20 < displayTrials.last_number
+  if (shuffleFlag) {
+    setShuffleTrial(displayTrials.last_number)
+  }
```

Comment:
Could you explain the purpose of this?

Reply:
Sorry, it is mistaken. This is a trial function that avoids giving bias due to position.